Makefile

```diff
@@ -1,45 +1,59 @@
-# Exclude vendor folder from project subfolders
-SUBDIRS = $(filter-out vendor/., $(wildcard */.))
-SUBCLEAN = $(addsuffix .clean,$(SUBDIRS))
-SUBTEST = $(addsuffix .test,$(SUBDIRS))
-SUBLINT = $(addsuffix .lint,$(SUBDIRS))
-SUBINSTALL = $(addsuffix .install,$(SUBDIRS))
-SUBBUILD = $(addsuffix .build,$(SUBDIRS))
-
-# invoke make all for all subprojects
-all: $(SUBDIRS)
-
-$(SUBDIRS):
-	$(MAKE) -C $@
-
-# invoke make clean for all subprojects
-clean: $(SUBCLEAN)
-
-$(SUBCLEAN): %.clean:
-	$(MAKE) -C $* clean
-
-# invoke make test for all subprojects
-test: $(SUBTEST)
-
-$(SUBTEST): %.test:
-	$(MAKE) -C $* test
-
-# invoke make lint for all subprojects
-lint: $(SUBLINT)
-
-$(SUBLINT): %.lint:
-	$(MAKE) -C $* lint
-
-install: $(SUBINSTALL)
-
-$(SUBINSTALL): %.install:
-	$(MAKE) -C $* install
-
-build: $(SUBBUILD)
-
-$(SUBBUILD): %.build:
-	$(MAKE) -C $* build
-
-.PHONY: all $(SUBDIRS) clean $(SUBCLEAN) test $(SUBTEST) lint $(SUBLINT) \
-	install $(SUBINSTALL) build $(SUBBUILD)
+IMAGE_NAME=dcos-log-dev
```

Comment:
I understand that changes to the Makefile are necessary when removing the extra `dcos-log` directory, but alot has changed in this file that is unrelated to this change. It would be better to make the change incrementally (with separate commits) so that I can easily review this and be *sure* that it does what's expected.

Reply:
This was the Makefile from subdirectory